README.md

```diff
@@ -44,7 +44,13 @@ Optional dependencies:
 - ZSTD library
 - OPENSSL library (signature handling in modinfo)
 
-Typical configuration:
+Typical configuration and installation
+
+    meson setup builddir/
+    meson install -C builddir/
```

Comment:
But why would you call install without building? If you are installing, unless you pass a destdir you will need to elevate privileges

Reply:
> `meson install` will invoke `meson compile`, similar to how `make install` kicks off `make`.

right, but I wouldn't mix then... `make && make install` means 2 operations with the second being up to the user to elevate privileges. 90% of the time you don ´t need to install neither and running from the build dir is fine.

Checking a few other, they also split the build and install phases:
https://docs.mesa3d.org/install.html#building-with-meson
https://github.com/systemd/systemd/blob/main/README#L241
https://github.com/mavlink-router/mavlink-router?tab=readme-ov-file#build  (heheh, of course it would since I maintain it , but the first 2 should be sufficient:))